change/@uifabric-date-time-2019-10-23-14-57-44-lorejoh12-fixWorkWeekCalendarHover.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
+  "comment": "updating work week hover state to match the actual days that are going to be selected. The mouse over state still takes the whole week into account",
+  "packageName": "@uifabric/date-time",
+  "email": "jolore@microsoft.com",
+  "commit": "81548bf2b1f2092d423578a5a7a8261317a46059",
+  "date": "2019-10-23T21:57:44.281Z",
+  "file": "C:\\Users\\jolore\\git\\office-ui-fabric-react\\change\\@uifabric-date-time-2019-10-23-14-57-44-lorejoh12-fixWorkWeekCalendarHover.json"
```

Comment:
The beachball tool should have been updated by now to produce change files without this file name. Would you mind getting rid of this line manually for now?

Reply:
I've been sitting on this PR for quite a while, thanks for the heads up!